src/components/Header/SearchBar/SuggestionBox.tsx

```diff
@@ -40,8 +41,10 @@ export const SuggestionBox = ({ results, isActive, error, query }: Props) => {
     return null;
   }
 
+  const containerStyle = displayLocation === 'homepage' ? { maxWidth: '100%' } : null;
+
   return (
-    <div aria-label="suggestions" className={container}>
+    <div aria-label="suggestions" className={container} style={containerStyle}>
```

Comment:
you could use `cn` here, I'm just not sure on the server-side rendering effect...

~~~js
<div className={cn(container, {'max-w-full': displayLocation === 'homepage'})}>
~~~